packages/fixie/src/main.ts

```diff
@@ -179,14 +179,36 @@ sources
   .description('Add a source to a corpus.')
   .option('--max-documents <number>', 'Maximum number of documents to crawl')
   .option('--max-depth <number>', 'Maximum depth to crawl')
+  .option('--include-patterns <pattern...>', 'URL patterns to include in the crawl')
+  .option('--exclude-patterns <pattern...>', 'URL patterns to exclude from the crawl')
   .action(
     async (
       corpusId: string,
       startUrls: string[],
-      { maxDocuments, maxDepth }: { maxDocuments?: number; maxDepth?: number }
+      {
+        maxDocuments,
+        maxDepth,
+        includePatterns,
+        excludePatterns,
+      }: { maxDocuments?: number; maxDepth?: number; includePatterns?: string[]; excludePatterns: string[] }
     ) => {
+      if (!includePatterns) {
+        term.yellow('Warning: ')(
+          'No --include-patterns specfied. This is equivalent to only crawling the URLs specified as startUrls.\n'
+        );
+        term.yellow('Warning: ')('Use ').red("--include-patterns '*'")(
```

Comment:
This is incorrect.  It should be **.  Apify has a "test glob" button you can use to experiment.

Reply:
Ah, thanks.